src/go/rdctl/cmd/root.go

```diff
@@ -159,6 +159,8 @@ func processRequestForUtility(response *http.Response, err error) ([]byte, error
 			break
 		case 401:
 			return nil, fmt.Errorf("user/password not accepted")
+		case 413:
+			return nil, fmt.Errorf("%d Payload Too Large: %s", response.StatusCode, response.Status)
```

Comment:
Nitpicking, but all other errors are all-lowercase:

```suggestion
			return nil, fmt.Errorf("%d payload too large: %s", response.StatusCode, response.Status)
```

Reply:
That's because HTTP errors are typically given as `[45]xx Capitalized Phrase` while general go error message generation is to assume that your message will be prefixed by some phrase (like `Error trying to open file: ...`) so you shouldn't capitalize it. I'd rather leave it as is.

Reply:
> That's because HTTP errors are typically given as [45]xx Capitalized Phrase 

Then why are your 401 and 500 status messages all lowercase? I was just commenting on consistency between the different HTTP status messages and not about Go errors at all.